typings/index.d.ts

```diff
@@ -11,8 +10,10 @@ declare module 'ytdl-core' {
       requestOptions?: {};
     }
 
+    type ChooseFormatQuality = 'lowest' | 'highest' | 'highestaudio' | 'lowestaudio' | 'highestvideo' | 'lowestvideo'
+
     interface chooseFormatOptions {
-      quality?: 'lowest' | 'highest' | 'highestaudio' | 'lowestaudio' | 'highestvideo' | 'lowestvideo' | string | number | string[] | number[];
+      quality?: ChooseFormatQuality | Omit<string, ChooseFormatQuality> | number | (ChooseFormatQuality | Omit<string, ChooseFormatQuality>)[] | number[];
```

Comment:
Can be make this a bit more beautiful with sth like?
```ts
type SingleOrArray<T> = T | T[];
```

other than that, although i think this is more of a interpreter/linter bug, i don't see any harm in merging this

Reply:
I made it thought:
```ts
type ExtendString<T extends string> = T | Omit<string, T>
...
interface chooseFormatOptions {
      quality?: ExtendString<ChooseFormatQuality> | number | ExtendString<ChooseFormatQuality>[] | number[]
...
```
What do you think? Maybe the name could be better, but that was the best that I thought

Reply:
i didn't even think of that 👍